pages/index.js

```diff
@@ -0,0 +1,25 @@
+import Blog from "../components/section/Blog";
+import Footer from "../components/section/Footer";
+import Project from "../components/section/Project";
+import Header from "../components/layout/Header";
+import Expertise from "../components/section/Expertise";
+import About from "../components/section/AboutSection/About";
```

Comment:
you can create an index.jsx file inside the sections folder that exports the components the import it in the index.jsx to reduce the imports